app/helpers/show_field_display_field_helper.rb

```diff
@@ -13,4 +13,25 @@ def is_excepted_dynamic_field?(field_config, document)
     (field_config.except || []).include? field_config.field
   end
 
+  # return the Blacklight::AbstractRepository in scope
+  def doc_repository
+    if defined?(:controller)
```

Comment:
This pattern appears in several places as a consequence of the fact that a helper may be included in a controller, but the related Blacklight helper scaffolding is not (and cannot be). So when these methods are called from a controller with Blacklight mixins, `repository` needs to be called directly.